### PR TITLE
A custom passport client modal can now be set via the config file

### PR DESCRIPTION
**Pull request by seriousjelly**

### CODE REVIEW

**Comment by seriousjelly** [comment]
#9

File: config/janitor.php [hunk 1]
```diff
@@ -37,6 +37,7 @@
 
         'passport' => [
             'oauth_token_url' => env('JANITOR_OAUTH_TOKEN_URL', 'http://localhost/oauth/token'),
+            'client_model' => \Laravel\Passport\Passport::class
```
Comment (pactode) [inline config/janitor.php]: This should be `\Laravel\Passport\Client::class`, also make sure to add the trailing comma required by StyleCI.

**Review by pactode — CHANGES_REQUESTED** [review]
Thanks! There's only a couple of things that would need to be changed. Can you do that and I'll merge it in?

File: config/janitor.php (hunk 1 above)
Comment (seriousjelly) [reply config/janitor.php]: Thanks. Pushed up those changes and passes tests.

**Review by pactode — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**